src/main/resources/templates/mail/daily-mail.html

```diff
@@ -0,0 +1,152 @@
+<!doctype html>
```

Comment:
W zadaniu chodziło o reużycie istniejącego szablonu przy pomocy występujących w nim m. in. konstrukcji warunkowych.
Ty dodałeś nowy plik szablonu (zasadniczo ma to sens, ale w zadaniu chodziło przećwiczenie możliwości ThymeLeaf ;)
Dodatkowo zauważ, że w obecnym rozwiązaniu pewne wartości ustawiasz "na sztywno" (np. `is_friend` czy `show_button`), nie ma więc najmniejszego sensu, aby w dedykowanym szablonie była przewidziana sytuacja odwrotna.